src/components/admin/Billing/index.tsx

```diff
@@ -234,7 +172,9 @@ function AdminBilling({ showAddPayment }: AdminBillingProps) {
                     <CardWrapper
                         height={invoiceCardHeight}
                         message={
-                            hydrated && selectedInvoice ? (
+                            active || !hydrated ? (
+                                <FormattedMessage id="admin.billing.label.lineItems.loading" />
+                            ) : selectedInvoice ? (
```

Comment:
Adding in `active` check to show loading to cover more cases when loading should be shown.